src/schema/items/acq.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: Acquisition
```

Comment:
am I catching it right that filename pretty much specifies the `altname` (#588)?